lib/amazon-pricing/definitions/database-type.rb

```diff
@@ -251,6 +251,16 @@ def self.operation_sf?(operation_name,multiaz=false)
       return false unless display_name  # unknown operation is presumed non sf
       self.database_sf?(display_name)
     end
+    # example: databases_sf() returns [MYSQL_STANDARD, ... ]
+    # Returns BOOL if operation string is RDS SF
+    # params: none
+    def self.databases_sf
+      dbs = []
```

Comment:
@@DB_ENGINE_MAP.each_with_object([]) do |(k,v), m|
 m << k if v[:sizeflex]
end

up to you, but more ruby-like